onnxruntime/core/framework/allocatormgr.h

```diff
@@ -11,21 +11,30 @@ namespace onnxruntime {
 
 using DeviceAllocatorFactory = std::function<std::unique_ptr<IDeviceAllocator>(OrtDevice::DeviceId)>;
 
+// TODO why does DeviceAllocatorRegistrationInfo have arena related configs?
+// TODO even if it should, they should be inside their own struct (OrtArenaCfg) as opposed to
+// littering them as individual members of DeviceAllocatorRegistrationInfo
```

Comment:
What's the reason for not using OrtArenaCfg here?

Reply:
This will cause a lot of unnecessary churn in this PR and the original intent will be lost. Created work item to address in a separate PR. https://msdata.visualstudio.com/Vienna/_workitems/edit/863400